modules/sdk-coin-avaxp/src/iface.ts

```diff
@@ -21,18 +21,18 @@ export interface TxData {
   inputs: TransferableInput[];
   memo: string;
   // addDelegator extends BaseTx
-  validator?: {
-    nodeID: string;
-    startTime: number; // unix time
-    endTime: number; // unix time
-    weight: number; // amount delegator stakes
-  };
-  stake?: {
-    lockedOuts: TransferableOutput[]; // array of transferable outputs that are locked during staking period
-  };
-  rewards_owner?: SECPOwnerOutput;
-  // addValidator extends BaseTx, validator, stake, rewards_owner
-  shares?: number; // 10,000 x percentage of reward taken from delegators
+  // validator?: {
```

Comment:
depending on transactionType -- baseTx, stakingTx, delegatorTx, and eventually import and exports the txData will be different.  Once we are in position to implement explainTransaction this will be uncommented.